### PR TITLE
General improvements and ws support

### PR DESCRIPTION
**Pull request by i1i1**
Closes https://github.com/hyperledger/iroha/issues/1137.

Signed-off-by: i1i1 <vanyarybin1@live.ru>

### CODE REVIEW

File: iroha_python/iroha2/__init__.py [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
 
     def submit_inst(self, inst):
-        return self.cl.submit_all_with_metadata([inst.to_rust()], {})
+        return self.submit_tx([inst])
+
+    def submit_tx_blocking(self, tx: list):
+        tx = [i.to_rust() for i in tx]
+        return self.cl.submit_all_blocking_with_metadata(tx, {})
+
+    def submit_inst_blocking(self, inst):
```
Comment (eadventurous) [inline iroha_python/iroha2/__init__.py:26]: Let's use full name for `instruction`, sorry for not telling it earlier

File: iroha_python/test.py [hunk 2]
```diff
@@ -12,6 +12,6 @@
 cl = Client(cfg)
 
 domain = Domain("xor")
-register = Instruction(RegisterBox(Expression(Value(IdentifiableBox(domain)))))
+register = Instruction(Register(Expression(Value(Identifiable(domain)))))
 
 cl.submit_inst(register)
```
Comment (eadventurous) [inline iroha_python/test.py:17]: Maybe it is better to also create a test for events?

Comment (i1i1) [reply iroha_python/test.py:17]: Check out `ws.py` file

File: iroha_python/iroha2/__init__.py (hunk 1 above)
Comment (i1i1) [reply iroha_python/iroha2/__init__.py:26]: For function names, right? May be we can rename it to `isi`?

File: iroha_python/test.py (hunk 2 above)
Comment (eadventurous) [reply iroha_python/test.py:17]: But it does not seem like a test, nothing is checked there. We should add some assertions.

File: iroha_python/ws.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+if __name__ == "__main__":
+    cfg = json.loads(open("./config.json").read())
+    cl = Client(cfg)
+    filter = EventFilter.Pipeline(pipeline.EventFilter(
+        entity=pipeline.EntityType.Transaction(),
+        hash=None,
+    ))
+    listener = cl.listen(filter)
+    while True:
+        # Strange thing -- ws doesnt work as an iterator
```
Comment (eadventurous) [inline iroha_python/ws.py:18]: I think we need to fix it, maybe you forgot to add `__iter__` or something else? Please consult with Ruslan on this.

Comment (i1i1) [reply iroha_python/ws.py:18]: The thing is that pyo3 is working weirdly. I guess we can wrap it with python class on python side.

**Review by Revertron — APPROVED** [review]

**Review by Revertron — APPROVED** [review]

**Review by eadventurous — APPROVED** [review]

**Pull request merged**